### PR TITLE
Contrib script and instructions

### PR DESCRIPTION
**Pull request by wardpenney**

### CODE REVIEW

**Comment by whmii** [comment]
lgtm

**Comment by tysongach** [comment]
:+1:

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -36,3 +36,15 @@ Some things that will increase the chance that your pull request is accepted:
...
 * Blank lines between rules.
 * No trailing whitespace. Blank lines should not have any space.
+
+**Contrib examples**
+
+For user-facing changes, please provide an example in sample page contained in
+the `./contrib` folder. Also, please put a screenshot of the changes into the
+Pull Request.
+
+To see the `./contirb` example page:
```
Comment (tysongach) [inline CONTRIBUTING.md]:
```diff
- ./contirb
+ ./contrib
```

Comment (wardpenney) [inline CONTRIBUTING.md]: Ahh good eye!

**Pull request merged**